src/giskardpy/qp/qp_solver_qpalm.py

```diff
@@ -84,6 +87,12 @@ def __init__(self, weights: cas.Expression, g: cas.Expression, lb: cas.Expressio
 
         self.free_symbols_str = [str(x) for x in free_symbols]
 
+        self.init_manipulability_variables(constraint_jacobian, grad_traces)
+        if self.use_manipulability:
```

Comment:
why not combine with the other stacked compiled function?

Reply:
If I do that shouldn't the other two stacked compiled functions be merged together as well? Or with which one should I combine mine?